### ISSUE TITLE
Obsoletion + NTR: adhesive extracellular matrix(GO:0062130) to Drosophila adhesive extracellular matrix

### ISSUE THREAD

**Issue by DaiqingChen** [geneontology/go-ontology#29499]
Please provide as much information as you can: 

* **GO term ID label:**
Replace ID label Adhesive extracellular matrix (GO:0062130) by "Drosophila adhesive extracellular matrix"

* **Request to add a taxon constraint:**
Drosophila melanogaster

* * **only in taxon: **
Drosophila melanogaster

* * **never in taxon: **

* **Request to remove a taxon constraint:**
The old term is misleading as in common usage the term "adhesive extracellular matrix" refers to the group of gene products that exert adhesive functions within the extracellular matrix (GO:0031012), while this term is actually defining a subset of drosophila gene products; the reference says "These glands synthesize and secrete massive amounts of a glue which attaches the pupae to the substrate during metamorphosis.", this is taxon-specific. 
We thus recommend changing the name to "drosophila adhesive extracellular matrix" or the synonym "puparial glue" and also change the parent term.

* **Change the parent to Specialized extracellular matrix (see #29265)** 

* **Supporting evidence if available (e.g PMID):**
Keep the same reference and definition: PMID:825230

<img width="1518" alt="Image" src="https://github.com/user-attachments/assets/3285190f-e474-420e-b243-1fc8558711f2" />

**Comment by Matrisome** [comment]
@DaiqingChen : Add link to slide deck and insert screenshot of revised slide 2

**Comment by cmungall** [comment]
Hi @DaiqingChen

You can find some of the original discussion that led to the creation of this term here:

- #17502

Interestingly, the original request from @hattrill was for a Drosophila-specific term, but after discussion we decided to introduce a very broad and general term.

Note also that it used to be common practice to attach the PMID that was used for the first annotation to the term definition, even if the PMID is much more specific. This is really confusing. We have since stopped this practice.

I agree with your broader analysis. The current term as it exists is **far too broad**, as it covers a huge range of secreted products that are different structurally and evolutionary. I support "repurposing" this term to be more specific as it was originally intended.

I think this is a good time to do some general cleanup. Let's look at some similar terms:

* GO BP term "puparial adhesion" https://amigo.geneontology.org/amigo/term/GO:0007594
* FlyBase gene group term "glue protein" https://flybase.org/reports/FBgg0001189.htm

The gene sets for the 3 terms are identical (modulo inclusion of Sgs9, which was purported to exist in 1976 but has not been genomically characterized)

It seems odd to me that we have 3 terms to describe the same set of genes, yet in GO we report "function unknown" for most of them. The biophysical function of these genes are understood in exquisite detail! The function for some of the genes is reported as "structural constituent of ECM" but this is inconsistently applied and IMO is pretty indirect.

I think the most conservative course of action here is to repurpose this term as you suggest in https://github.com/geneontology/go-ontology/issues/29265#issuecomment-2557670020 under "specialized ECM", relabel and redefine to be less generic, add a taxon constraint. I think the correct TC is NCBITaxon:480117 ! Cyclorrhapha (see https://link.springer.com/chapter/10.1007/978-3-319-40740-1_15) but putting "Dipteran" in the name is good enough for clarity. 

I would make an equivalence xref to FlyBase:FBgg0001189 such that gene sets are guaranteed to be the same (though this seems fairly stable)

And not to overload this ticket, but I'm questioning the utility of:

```yaml
id: GO:0007594
name: puparial adhesion
namespace: biological_process
def: "The adhesion of the puparia of Diptera to their substrate; normally effected by a 'glue' secreted by the larval salivary gland and expectorated at the time of pupariation." [GOC:ma]
synonym: "puparial glue" RELATED []
is_a: GO:0022609 ! multicellular organism adhesion to substrate
relationship: part_of GO:0007591 ! molting cycle, chitin-based cuticle
```

Is this truly a biological process, or more of a biophysical process? I think if we do want to represent the GO-BP aspect of glue genes we should have a term for the secretory pathway for generating them, which is well understood with defined starts and ends (though the glue genes themselves would not be annotated here). It seems odd to try and force the glue genes into a biological process.

I also wonder if we want to do a better job of gene products whose function is biophysical. We have the "structural constituent" branch but this seems to be applied consistently. 

(oh and the reference for the FBgg0001189 is Naba et al, we have come full circle :-)

**Comment by Matrisome** [comment]
@cmungall Thanks for the feedback! 
Your proposal seem reasonable. If we need additional insight, I can reach out to our ECM Drosophila expert.

**Comment by ValWood** [comment]
I'm guessing this doesn't require a taxon constrain. Can we close?

**Comment by cmungall** [comment]
I'd like to discuss this one further IMO we broadened the meaning of this
term too much, be good to discuss with Helen.

**Comment by ValWood** [comment]
tagging helen @hattrill

**Comment by ValWood** [comment]
For yeast , we have a set of genes that are involved in cell-cell adhesion (flocculins), and we use the term 
GO:0098631   cell adhesion mediator activity
is a function term like that useful here?

**Comment by hattrill** [comment]
Similar...the issue is that it is non-selective and is not cell to surface. GO:0005488 'binding' is decribed as selective, so such a term might just have to hang off GO:0003674 'molecular_function' which makes me nervous.

Something like:
external surface adhesive activity
The stable, non-selective binding by a protein to an external surface, mediating the organism's attachment to an environmental substrate. This includes proteins used by insects to anchor pupae to surfaces.

**Comment by ValWood** [comment]
I was thinking that  would be a useful grouping term for 

GO:0098631 cell adhesion mediator activity

but these drosophila 'hooks' seem quite a different thing  , more physical structure if I understand correctly.

For yeast the  glycosylation pattern, and length appears to provide the binding specificity

**Comment by ValWood** [comment]
@hattrill I see that's what you said (I missed the point about binding being selective)

**Comment by hattrill** [comment]
The hooks structural - part of the chitinous cuticle. This is for the glue proteins, they are sort of spat out and stick the pupa to a surface just like glue

**Comment by pgaudet** [comment]
Can we define action items: Is this correct: 

- [ ] obsolete GO:0062130 adhesive extracellular matrix: 8 fly annotations
- [ ] create a cell-substrate adhesion activity? although the current [cell adhesion mediator activity](http://purl.obolibrary.org/obo/GO_0098631) cover bother cell-cell and cell substrate adhesion. That seems specific enough for a MF ? (def = The binding by a cell-adhesion protein on a cell surface to an adhesion molecule on another cell surface or an external substrate, to mediate adhesion of the cell to the external substrate or to another cell.) 

Thanks, Pascale

**Comment by hattrill** [comment]
Yes: obsolete GO:0062130 adhesive extracellular matrix: 8 fly annotations

Don't think "[cell adhesion mediator activity](http://purl.obolibrary.org/obo/GO_0098631)" does it as it isn't really cell-to-surface, but more cuticle-to-surface and binding is non-specific

**Comment by pgaudet** [comment]
Dear all, 

The proposal has been made to obsolete GO:0062130 adhesive extracellular matrix. The reason for obsoletion is that it is too broadly defined. There are 8 fly annotations to this term (being removed). There are no mappings; this term is not in any subsets. 

You can comment on the  ticket: https://github.com/geneontology/go-ontology/issues/29499

Thanks, Pascale

**Comment by hattrill** [comment]
> Do you want a new MF term for this ? 'molecular glue activity'? (or something better?)

That would be good:
I suggested: bioadhesive activity

"The stable, non-selective binding by a protein to an external surface, mediating the organism's attachment to an environmental substrate. This includes proteins used by insects to anchor pupae to surfaces. PMID:39370426.

**Comment by pgaudet** [comment]
What would be the parent ? molecular adaptor activity?

**Comment by hattrill** [comment]
molecular adaptor activity would work

**Comment by cmungall** [comment]
```yaml
name: molecular adaptor activity
namespace: molecular_function
alt_id: GO:0032947
def: "The binding activity of a molecule that brings together two or more molecules through a selective, non-covalent, often stoichiometric interaction, permitting those molecules to function in a coordinated way." [GOC:mtg_MIT_16mar07, GOC:vw]
```

- The glue proteins are not permitting the molecules to function in a coordinated way, as per the definition. Molecular adaptor is presumably intended for functions in signaling pathways
- the interaction mechanism doesn't seem right, glue proteins are more of a macroscopic adhesive interaction whereas this definition fits more at the molecular level

For now, how about putting under

```yaml
id: GO:0005198
name: structural molecule activity
namespace: molecular_function
def: "The action of a molecule that contributes to the structural integrity of a complex." [GOC:mah, GOC:vw]
```

Also, if we add a term as general as bioadhesivity presumably we need to backfill it as a lot of cell adhesion proteins would fit underneath it

**Comment by pgaudet** [comment]
+name: bioadhesive activity
+namespace: molecular_function
+def: "A structural molecule ativity that mediates an organism's attachment to an environmental substrate via stable, non-selective binding by a protein to an external surface. Examples includes proteins used by insects to anchor pupae to surfaces." [PMID:39370426]
+is_a: GO:0005198 ! structural molecule activity
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29499" xsd:anyURI
+created_by: pg
+creation_date: 2025-01-23T07:41:17Z
